build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1029,6 +1029,19 @@ message ActionResult {
 
   // The details of the execution that originally produced this result.
   ExecutedActionMetadata execution_metadata = 9;
+
+  // An optional additional salt hash value used to place this `ActionResult`
+  // into a separate cache universe or "epoch" from other instances having
+  // the same field contents. This salt typically comes from operational
+  // configuration specific to sources such as repo and service configuration,
+  // and allows disowning an entire set of ActionResults that might have been
+  // poisoned by buggy suftware or tool failures.
+  //
+  // As an example for repo configuration, a repo could contain a special file
+  // such as "CacheEpoch.txt" containing an incrementing number, whose hash is
+  // added to this salt value. Repo owners could increment the number when
+  // needed to invalidate its action cache.
+  Digest action_salt = 10;
```

Comment:
Instead of going down this route, I actually thought that @EricBurnett's idea of moving this into Action made lots of sense. I suspect it's sufficient to just add a field like this?
```
bytes salt = ...;
```

Reply:
Good point. Updating.